pom.xml

```diff
@@ -265,12 +282,17 @@
         <dependency>
             <groupId>com.googlecode.owasp-java-html-sanitizer</groupId>
             <artifactId>owasp-java-html-sanitizer</artifactId>
-            <version>20170515.1</version>
+            <version>20171016.1</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
```

Comment:
why?

Reply:
It pulls in a conflicting version - See above

Reply:
same